extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -45,8 +57,43 @@ export class UserController {
     public userService: UserService<User, Credentials>,
     @inject(SecurityBindings.USER, {optional: true})
     private user: UserProfile,
+    @inject(UserServiceBindings.USER_REPOSITORY)
+    public userRepository: UserRepository
   ) {}
 
+  @post('/signup', {
```

Comment:
Personally I'd prefix with `users` resulting in `/users/signup` endpoint for clarity.  Not essential, to change.  Up to you.